### PR TITLE
Adding info about running along with a web app project

### LINKED ISSUE #5: Does this buildpack require a dedicated Heroku project?
It is not clear from the `readme.md` whether this buildpack requires a dedicated, empty Heroku project. The following line:

> On your machine create a new directory for the VividCortex agents project.

…seems to suggest that, but it is still unclear whether it would work to just add the buildpack to an existing Cedar-14 app.

For the record, adding the buildpack to an existing app doesn't seem to work. Upon deploy, the buildpack is fetched, but no VividCortex app is detected, so the buildpack does not install the required files (`bin/compile` et al are not even run).

Please clarify, in the readme, whether a dedicated Heroku project/app is required, and if not, advise how to add this to an existing project.

### PR DESCRIPTION
**Pull request by AgentO3**
Adds documentation about running agents along with a project with a web app.

Closes #5

### CODE REVIEW

**Comment by AgentO3** [comment]
@gkristic looks good?

File: readme.md [hunk 1]
```diff
@@ -4,10 +4,12 @@ Installs VividCortex agents in a Heroku dyno.
...
+This buildpack installs VividCortex agents as part of the dyno build process. This works with PostgreSQL and MySQL, provided that pg_stat_statements or performance_schema, respectively, are enabled. Note that for PostgreSQL versions 9.2 and later it's enabled by default. This will not work with the "hobby" level Heroku PostgreSQL database offerings because they do not have "pg_stat_statements" enabled. We also support Redis and MongoDB, but only for showing charts. There's no query capture because there's no pg_stat_statements or performance_schema equivalent for them. For more information about the nuances of using agents for off-host monitoring refer to [the documentation here](https://vividcortex.com/docs/installing/#off-host-monitoring).
 
 ### Setup
 
+In this setup tutorial we will be using a dedicated Heroku project to run our agents in. By default Heroku only allows one buildpack per project. If you wish to configure agents to run in a project with a web application you can do so by using [the buildpack-multi](https://github.com/ddollar/heroku-buildpack-multi)buildpack. This will allow you to define more than one build pack for your project.
```
Comment (gkristic) [inline readme.md]: Missing space after the link.

**Comment by gkristic** [comment]
:+1:

**Comment by attilaolah** [comment]
Awesome, thanks for clarifying this!

:bowtie:

**Pull request merged**